dmoj/urls.py

```diff
@@ -327,39 +326,39 @@ def paged_list_view(view, name):
         ])),
     ])),
 
-    url(r'^feed/', include([
-        url(r'^problems/rss/$', ProblemFeed(), name='problem_rss'),
-        url(r'^problems/atom/$', AtomProblemFeed(), name='problem_atom'),
-        url(r'^comment/rss/$', CommentFeed(), name='comment_rss'),
-        url(r'^comment/atom/$', AtomCommentFeed(), name='comment_atom'),
-        url(r'^blog/rss/$', BlogFeed(), name='blog_rss'),
-        url(r'^blog/atom/$', AtomBlogFeed(), name='blog_atom'),
+    re_path(r'^feed/', include([
```

Comment:
Can be `path`, like all the plain string `include` cases.

Reply:
Thank you for your review. I'll fix the code more.